src/components/reusable/table/table-head.ts

```diff
@@ -15,6 +15,23 @@ import styles from './table-head.scss';
 export class TableHead extends LitElement {
   static override styles = [styles];
 
+  override firstUpdated() {
+    this.addEventListener('on-sort-changed', this.handleChildSort as EventListener);
+  }
```

Comment:
Add event listeners in `connectedCallback()` instead. Also need to remove event listeners in `disconnectedCallback()`, otherwise they can start to duplicate themselves.

Reply:
Right, I forgot the connectedCallback. Was still thinking in React.